wgpu-types/src/math.rs

```diff
@@ -0,0 +1,57 @@
+//! Utilitary math functions.
+
+///
+/// Aligns a `value` to an `alignment`.
+///
+/// Returns the first number greater than or equal to `value` that is also a
+/// multiple of `alignment`. If `value` is already a multiple of `alignment`,
+/// `value` will be returned.
+///
+/// # Examples
+///
+/// ```
+/// # use wgpu_types::math::align_to;
+/// assert_eq!(align_to(253, 16), 256);
+/// assert_eq!(align_to(256, 16), 256);
+/// assert_eq!(align_to(0, 16), 0);
+/// ```
+///
+pub fn align_to(value: u32, alignment: u32) -> u32 {
```

Comment:
Any reason this can't remain generic? I know it wouldn't get the po2 optimization, but this isn't likely to be a very hot function